### PR TITLE
Add gh action to run tests

### PR DESCRIPTION
**Pull request by anarute**
Add action to be triggered on every pull request and run the automated tests.

### CODE REVIEW

**Comment by anarute** [comment]
@jaragunde now the automated tests will run on every PR 🎉

File: composer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "igalia/phpreport",
     "description": "phpreport",
     "type": "project",
-    "license": "GNU GPLv3",
+    "license": "LGPL-3.0-or-later",
```
Comment (jaragunde) [inline composer.json:5]: There are a few imported lines of LGPL code, but the whole application is distributed under GPL terms. Is there a problem with "GPL-3.0-or-later"?

**Review by jaragunde — COMMENTED** [review]
This is a great step towards code reliability, thanks! I just have a nit regarding the license.

File: composer.json (hunk 1 above)
Comment (anarute) [reply composer.json]: No problem, I've just changed it to GPL-3.0-or-later

**Pull request merged**